### PR TITLE
Update log drains pricing

### PR DESCRIPTION
**Pull request by jordienr**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

### CODE REVIEW

File: apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx [hunk 1]
```diff
@@ -446,13 +460,41 @@ export function LogDrainDestinationSheetForm({
...
+                      Documentation <ExternalLink className="w-4 h-4" />
+                    </Link>
+                  </div>
+                </div>
+              }
+            >
+              <ul className="text-right text-foreground-light">
+                <li className="text-brand-link text-base">$60 / drain / month</li>
+                <li>$0.20 per million events</li>
+                <li>$0.09 per GB</li>
```
Comment (Ziinc) [inline apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx:487]:
```suggestion
                <li>$0.09 per GB egress</li>
```

File: apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx [hunk 2]
```diff
@@ -446,13 +460,41 @@ export function LogDrainDestinationSheetForm({
...
+                      href="https://supabase.com/docs/guides/platform/log-drains"
+                    >
+                      Documentation <ExternalLink className="w-4 h-4" />
+                    </Link>
+                  </div>
+                </div>
+              }
+            >
+              <ul className="text-right text-foreground-light">
+                <li className="text-brand-link text-base">$60 / drain / month</li>
```
Comment (Ziinc) [inline apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx:485]:
```suggestion
                <li className="text-brand-link text-base">$60 per drain per month</li>
```

Comment (Ziinc) [reply apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx:485]: @kevcodez any preference? went with what is in the pricing page

**Review by Ziinc — COMMENTED** [review]
copy wise would be good if @kevcodez had a look for cross-app consistency

File: apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx (hunk 2 above)
Comment (kevcodez) [reply apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx:485]: No hard feelings - given this is inside of a modal, can also use the shorter form with a slash

**Review by alaister — APPROVED** [review]
LGTM!

- Viewed on preview

**Pull request merged**